model-optimizer/extensions/front/RollWithEmptyAxesReplacer_test.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import unittest
+
+from extensions.front.RollWithEmptyAxesReplacer import RollWithEmptyAxesReplacer
+from mo.front.common.partial_infer.utils import int64_array
+from mo.graph.graph import Node
+from mo.utils.ir_engine.compare_graphs import compare_graphs
+from mo.utils.unittest.graph import build_graph, const, result, regular_op
+
+nodes_attributes = {
+    **regular_op('placeholder', {'type': 'Parameter'}),
+    'roll': {'type': 'Roll', 'kind': 'op', 'op': 'Roll', 'name': 'roll', 'axes': int64_array([-1, 2, 3]),
```

Comment:
why you don't use regular op everywhere?

Reply:
Changed to regular_op everywhere.